### DISCUSSION TITLE
Consider reducing the size of app.js

### DISCUSSION THREAD

**Discussion by LittleHans8** [filamentphp/filament#5043] [Ideas]
Consider reducing the size of app.js, which contains some redundant translated text, resulting in an oversized app.js. For example, for the DateTimePicker component I only need the English translation, but app.js contains translations for other languages. The same problem occurs with the File upload component.

<img width="973" alt="image" src="https://user-images.githubusercontent.com/15800501/203476194-d4f165fa-0168-4c7e-9cdc-ddfd7e6003ef.png">

<img width="795" alt="image" src="https://user-images.githubusercontent.com/15800501/203476837-ae36bc36-64db-48e3-afab-3a5aa6762684.png">

<img width="934" alt="image" src="https://user-images.githubusercontent.com/15800501/203476643-d53a2986-f5fd-4261-aaf2-3c214c9f5a9d.png">

**Comment by danharrin** [comment]
Fixed in v3, form components are loaded individually when they are requested
